src/sassAutocomplete.ts

```diff
@@ -170,13 +211,15 @@ class SassCompletion implements CompletionItemProvider {
 
     let atRules = [],
       properties = [],
-      values = [];
+      values = [],
+      variables = [];
 
     if (value) {
       values = getValues(cssSchema, currentWord);
     } else {
       atRules = getAtRules(cssSchema, currentWord);
       properties = getProperties(cssSchema, currentWord, config.get('useSeparator', true));
+      variables = getVariables();
```

Comment:
Moving the `variables` variable out of the `if` fixes the no-complete issue. I have done this.